contrib/clojure-package/examples/cnn-text-classification/lein-cljfmt-check

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
```

Comment:
What do you think about moving this script to a bit more top-level directory and then applying it recursively?

Reply:
@marcoabreu 
I did not think of this, but yes this is a great idea indeed.